### ISSUE TITLE
Archive?

### ISSUE THREAD

**Issue by spwilson2** [UW-UPL/UPL-Website-Revamp#1]
What is this repo for? I'm guessing this is the old upl site. Can we archive this?

**Comment by LOZORD** [comment]
I think either renaming it or sticking it in the archive repo might be good
ideas.

**Comment by spwilson2** [comment]
Yeah I don't like having a repo around that we aren't using when we already
have an archive repo

**Comment by spwilson2** [comment]
I'll move this into the archive.

**Comment by katamaritaco** [comment]
Yeah I can archive this now.
